wiki/osu!_File_Formats/Osu_(file_format)/en.md

```diff
@@ -80,7 +80,7 @@ These options are only relevant when opening maps in the [beatmap editor](/wiki/
 | `CircleSize` | Decimal | CS setting (0–10) |
 | `OverallDifficulty` | Decimal | OD setting (0–10) |
 | `ApproachRate` | Decimal | AR setting (0–10) |
-| `SliderMultiplier` | Decimal | Base slider velocity in hecto-[osu! pixels](/wiki/osupixel) per beat <!-- TODO: this prefix sounds awful ಠ_ಠ --> |
+| `SliderMultiplier` | Decimal | Base slider velocity in hundred-[osu! pixels](/wiki/osupixel) per beat |
```

Comment:
the solution is simple

```suggestion
| `SliderMultiplier` | Decimal | Base slider velocity in 100 [osu! pixels](/wiki/osupixel) per beat |
```

Reply:
isn't this the unit prefix? like `hectometre` =  hundred of metre?

Reply:
Has this not been changed?

Reply:
still not sure whether to use 100 or hundreds of